packages/config/src/projects/bobanetwork.ts

```diff
@@ -0,0 +1,204 @@
+        description:
+          'Ultimately Boba Network will use fraud proofs to enforce state correctness. This feature is currently disabled and the system permits invalid state roots.',
+        risks: [
+          {
+            category: 'Funds can be stolen if',
+            text: 'an invalid state root is submitted to the system.',
+          },
+        ],
+        references: [
+          {
+            text: 'The incentive contract for verification proofs is disabled - Boba FAQ',
+            href: 'https://docs.boba.network/faq#the-incentive-contract-for-verification-proofs-is-disabled',
+          },
+        ],
+      },
+      dataAvailability: {
+        ...DATA_AVAILABILITY.ON_CHAIN_CANONICAL,
+        references: [
+          {
+            text: 'Data Availability Batches - Paradigm Research',
+            href: 'https://research.paradigm.xyz/optimism#data-availability-batches',
+          },
+        ],
+      },
+      operator: {
+        ...OPERATOR.CENTRALIZED_SEQUENCER,
+        references: [
+          {
+            text: 'Boba operates the only "Sequencer" node - Boba FAQ',
+            href: 'https://docs.boba.network/faq#boba-operates-the-only-sequencer-node',
+          },
+        ],
+      },
+      forceTransactions: {
+        ...FORCE_TRANSACTIONS.CANONICAL_ORDERING,
+        references: [
+          {
+            text: 'Chain Contracts - Boba documentation',
+            href: 'https://docs.boba.network/developer-docs/chain-contracts',
+          },
+        ],
+      },
+      exitMechanisms: [
+        {
+          name: 'Regular exit',
+          description:
+            'When a user initiates a withdrawal it is processed as a L2 to L1 message. Because Boba Network is an optimistic rollup this transaction has to be included in a block and finalized. This takes several days to happen after which the funds can be withdrawn on L1.',
+        },
+        {
+          name: 'Fast exit',
+          description:
+            'Users can initiate a fast exit which makes use of liquidity pools, and charges a small fee for the convenience. Users funds can then be withdrawn on L1 after only minutes.',
+        }
+      ],
+      smartContracts: {
+        name: 'EVM compatible smart contracts are supported',
+        description:
+          'Boba Network uses the Optimistic Virtual Machine (OVM) to execute transactions. This is similar to the EVM, but is independent from it and allows fraud proofs to be executed.',
+        risks: [
+          {
+            category: 'Funds can be lost if',
+            text: 'there are mistakes in the highly complex OVM implementation.',
+          }
+        ],
+        references: [
+          {
+            text: 'Execution Contracts - Learn more about the Boba Network execution contracts',
+            href: 'https://docs.boba.network/developer-docs/chain-contracts/execution-contracts',
+          },
+        ],
+      },
+      contracts: {
+        addresses: [
```

Comment:
Please include all contracts from this list: https://docs.boba.network/developer-docs/mainnet-addresses

Reply:
Added.